app/models/product.rb

```diff
@@ -55,10 +58,18 @@ def discount_price_cannot_be_greater_than_price_value
   end
 
   def split_description_to_words
```

Comment:
please rebase with the master branch. I can see merge conflicts.
You can discuss if you have any doubts related to rebasing